back/spec/services/survey_results_generator_service_spec.rb

```diff
@@ -584,30 +584,59 @@
       end
 
       context 'with grouping' do
-        let(:grouped_linear_scale_answers) do
-          [
-            { answer: 5, count: 1, groups: [
-              { count: 1, group: nil }
-            ] },
-            { answer: 4, count: 1, groups: [
-              { count: 1, group: 'la' }
-            ] },
-            { answer: 3, count: 8, groups: [
-              { count: 1, group: 'ny' },
-              { count: 7, group: nil }
-            ] },
-            { answer: 2, count: 5, groups: [
-              { count: 5, group: nil }
-            ] },
-            { answer: 1, count: 2, groups: [
-              { count: 2, group: nil }
-            ] },
-            { answer: nil, count: 5, groups: [
-              { count: 1, group: 'la' },
-              { count: 3, group: 'other' },
-              { count: 1, group: nil }
-            ] }
-          ]
+        let(:grouped_linear_scale_results) do
```

Comment:
Grouped linear scales were actually not working correct, which I didn't know because I was lazy when writing tests (was only testing the answers, not the whole response). Now the test is correct